packages/types/src/documents/app/automation/schema.ts

```diff
@@ -0,0 +1,257 @@
+import { SortOrder } from "../../../api"
+import { EmptyFilterOption, SearchFilters } from "../../../sdk"
+import { HttpMethod } from "../query"
+import { Row } from "../row"
+import {
+  AutomationActionStepId,
+  AutomationResults,
+  EmailAttachment,
+  LoopStepType,
+} from "./automation"
+
+export type AutomationStepInputMap = {
```

Comment:
We should use the same typing as `packages/server/src/automations/actions.ts`

Reply:
What do you mean by this, it's just a map so we can access the inputs dynamically?

Reply:
I meant to type the object to something like `const ACTION_IMPLS: {
  [K in AutomationActionStepId]?: (opts: AutomationStepInput<K>) => Promise<any>
} =`